composer.json

```diff
@@ -13,13 +13,14 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "vlucas/valitron": "~1.1",
-        "doctrine/dbal": "^2.5.4",
-        "sabre/event": "~2.0"
+        "php": ">= 8.1.11",
```

Comment:
If the PHP version is going to be bumped up to PHP8+, then we will have to do a major version release since this is a huge breaking change.

Reply:
Hi @vlucas It looks like I did something wrong with the branch and repository I pushed to. Especially the last commit today was addressed to my fork. The compatiblity for 8.1 nevertheless is given. Are you willing to make a release out of it?

Reply:
Yes, I can do a major release to support PHP8+.